sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/aio/_azureappconfigurationproviderasync.py

```diff
@@ -334,9 +307,33 @@ class AzureAppConfigurationProvider(Mapping[str, Union[str, JSON]]):  # pylint:
     keys. Enables resolution of Key Vault references in configuration settings.
     """
 
-    def __init__(self, endpoint, replica_client_manager, **kwargs) -> None:
+    def __init__(self, endpoint, **kwargs) -> None:
         self._origin_endpoint = endpoint
-        self._replica_client_manager = replica_client_manager
+
+        if "user_agent" in kwargs:
```

Comment:
code changes seem big enough to do a separate PR explaining why we're changing this.

Reply:
This is moving code from `_buildprovider` to inside the init.